core/src/main/java/org/keycloak/representations/idm/AbstractAuthenticationExecutionRepresentation.java

```diff
@@ -67,6 +67,9 @@ public void setPriority(int priority) {
      *
      * @return
      */
+    @Deprecated
+    private boolean autheticatorFlow;
```

Comment:
Instead of adding this unused field I would be happy to remove the following accessors, unfortunately, the latter is a breaking change for the import/export functionality.
cc. @stianst @pedroigor

Reply:
Here we are adding back the field, that remains unused, since it's emitted by the Json serialization.
If we don't add it, you would need to remove all the relevant fields `autheticatorFlow` (WITH the typo) from the exported `RealmRepresentation` before importing it as a CR.

Reply:
@stianst this is the only change in the core I'm leaving, as far as I can tell it should be ok, but let me know if I have to duplicate even this file.